electrical/pcb/control_board/readme.md

```diff
@@ -0,0 +1,107 @@
+## Components
```

Comment:
Hmm, I know this is a file that I added on my revF branch ....shouldn't be on this PR. What happened?

There are some other files like this as well on the PR. Think some cleanup is needed. 

At this point, what is the minimal set of files you need to change with this PR? I'd like to restrict it to only those files.

(let me know if it's easier for me to go in and clean up stuff, already had to do that quite a few times with PRs on this repo...)

Reply:
I based my changes on the revF branch because of what you said before. I guess I misunderstood. I will rebase my changes onto the master branch.